sr_logging_common/launch/sr_rosbag_log.launch

```diff
@@ -9,6 +9,9 @@
   <arg name="log_directory" default="."/>
   <arg name="log_bag_prefix" default="sr_log"/>
 
+  <arg name="core_dump_path" default="/home/user/.ros/log/core_dumps"/>
+  <arg name="desired_folder_size" default="1000000000"/>
```

Comment:
@biofotis is it not 1GB. It is 0.93 GB. Use this for calculations http://www.matisse.net/bitcalc/?input_amount=1000000000&input_units=bytes&notation=legacy